package.json

```
@@ -13,8 +13,8 @@
   "dependencies": {
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
-    "@iexec/dataprotector": "^0.0.0-nightly-8395225223034e4980135f6bf58f0a370721b496",
-    "@iexec/web3mail": "^0.0.0-nightly-cb75c44ed80f3698dd7923d62ccaf609a455305d",
+    "@iexec/dataprotector": "^0.1.0",
+    "@iexec/web3mail": "^0.0.0-nightly-e93b6b6eaa87abca06f8a829bb5b430423b10df8",
```

use fixed version when the version does not respect semver

```suggestion
    "@iexec/web3mail": "0.0.0-nightly-e93b6b6eaa87abca06f8a829bb5b430423b10df8",
```